From d351f2666afaa97437154b8a0e52b3557e186943 Mon Sep 17 00:00:00 2001
From: Chad Reese <kreese@caviumnetworks.com>
Date: Fri, 14 May 2010 14:19:38 -0700
Subject: [PATCH 186/337] MIPS: Octeon: Display the Core number in cache
 exceptions.

Based On SDK 3.0.0-482

Having the physical core number with a cache exception can be useful for debug
purposes.

Signed-off-by: Chad Reese <kreese@caviumnetworks.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
Signed-off-by: Jack Tan <jack.tan@windriver.com>
---
 arch/mips/mm/c-octeon.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/mm/c-octeon.c b/arch/mips/mm/c-octeon.c
index 44e69e7..979e4b1 100644
--- a/arch/mips/mm/c-octeon.c
+++ b/arch/mips/mm/c-octeon.c
@@ -293,7 +293,7 @@ static void  cache_parity_error_octeon(int non_recoverable)
 	unsigned long coreid = cvmx_get_core_num();
 	uint64_t icache_err = read_octeon_c0_icacheerr();
 
-	pr_err("Cache error exception:\n");
+	pr_err("Core%lu: Cache error exception:\n", coreid);
 	pr_err("cp0_errorepc == %lx\n", read_c0_errorepc());
 	if (icache_err & 1) {
 		pr_err("CacheErr (Icache) == %llx\n",
-- 
1.7.5.4

